guides/resource-provisioning/release-pipeline.md

```diff
@@ -0,0 +1,41 @@
+# Release pipeline
+
+Release pipelines are run from Jenkins and a deployment to the development environment is automatically triggered by the CI pipeline if building from a main branch.
+
+**Note** this approach to a release pipeline is only a temporary approach and will be replaced when our release capability using Azure DevOps is ready for use.
+
+## Configure release pipeline
+
+- navigate to your workstream folder in Jenkins
+- select `Credentials -> (global) -> Add Credentials`
+- select `Secret text` type and set both `Description` and `Id` to `<respository name>-deploy-token`, for example `ffc-demo-web-deploy-token`
+- set the secret value to be a unique string
```

Comment:
You mention this is only temporary so it might not be worth changing anything based on this suggestion. Either way, the suggestion is thus...
Having some additional context added to this bullet about what the string is and how it is used i.e. as a 'secret' token used for 'authenticating' (unnecessary quotes!) the triggering of the deploy job by the build job and the string is entered later in the deploy creation.